protob/messages.proto

```diff
@@ -349,6 +352,42 @@ message LoadDevice {
 }
 
 /**
+ * Request: Load Decred seed PGP word list from the computer
+ * @next ButtonRequest
+ * @next Success
+ * @next Failure
+ */
+message DecredLoadDevice {
```

Comment:
Did you need a new message or could you have added it as an enum in the existing message?